dashboard/app/assets/javascripts/components/report_abuse_form.jsx

```diff
@@ -0,0 +1,139 @@
+window.dashboard = window.dashboard || {};
```

Comment:
Yeah, probably. No linting in dashboard (yet), and I suspect there will be lots of issues like this if we turn it on, but might as well stay ahead of the curve :)